### DISCUSSION TITLE
A toolchain for converting tailwind styles to equivalent PandaCSS styles

### DISCUSSION THREAD

**Discussion by osdiab** [chakra-ui/panda#1180] [Ideas]
Hi all, I've been using [ShadCN](https://ui.shadcn.com/) to bootstap basic components, though the one annoying thing is all its styles are written in Tailwind so I end up needing to convert them all. I've convinced an LLM to do a considerable amount of the legwork, but I imagine it would be helpful for many developers moving over to have some tooling to help with such a migration. Thoughts?

**Comment by osdiab** [comment]
Oh, I actually saw that @astahmer made https://github.com/astahmer/tw2panda - this looks promising!

**Reply by astahmer** [reply]
yes ! a few weeks (months ?) ago I was myself in that process of converting different tailwind libs (including shadcn) to panda and this helped a lot. there's currently a CLI, online playground and a VSCode is ready to be published to the marketplace (working on that right now)
